client/views/network.tpl

```diff
@@ -1,5 +1,5 @@
 {{#each networks}}
-<section id="network-{{id}}" class="network" data-id="{{id}}" data-nick="{{nick}}" data-options="{{toJSON serverOptions}}">
+<section id="network-{{id}}" class="network name-{{slugify name}} net-{{slugify serverOptions.NETWORK}}" data-id="{{id}}" data-nick="{{nick}}" data-options="{{toJSON serverOptions}}">
```

Comment:
Nah because `name-xxx` is totally freeform. For instance, if I was connecting to IRCHighway I could name the network `irchighway`, `irch`, `rp1`, etc. Where in all those cases, the `name-xxx` would be different it would always have `net-irchighway` (since that's what's given in `RPL_ISUPPORT`).

I could see themes trying to, say, show an icon / use specific colours depending on the network, which is why I included both.